spec/imsc-hrm.html

```diff
@@ -13,10 +13,10 @@
 <script src="https://www.w3.org/Tools/respec/respec-w3c" class="remove" defer></script>
   <script class='remove'>
 var respecConfig = {
-                    specStatus:   "CR"
+                    specStatus:   "PR"
                 //,   processVersion: "2018"
                 //,   previousMaturity: "PR"
-                ,   publishDate: "2023-06-22"
+                ,   publishDate: "2024-01-30"
                 //,   modificationDate: "2020-04-27"
                 //,   errata: ""
                 ,   crEnd: "2023-07-20"
```

Comment:
At the moment the SOTD is showing an invitation to review and provide feedback by 2024-01-11, i.e. today! Not sure where that date derives from, maybe this crEnd, hopefully not though...

Reply:
I did not know what date to pick. @himorin Can you suggest one?

Reply:
If you have concerns about describing the IR status please let me know @himorin . I think it's clear that we have met the CR exit criteria. 

Otherwise if you can get a suggestion for a specific date that would be very helpful.